sql/semisync_master.cc

```diff
@@ -679,6 +672,11 @@ int Repl_semi_sync_master::report_reply_binlog(uint32 server_id,
       m_wait_file_name_inited = false;
     }
   }
+  if (rpl_semi_sync_master_enabled)
```

Comment:
I don't think this check is necessary, as the function preamble already checks `get_master_enabled()` and exits if not.